### PR TITLE
Multidb support

### PR DESCRIPTION
**Pull request by bartekrylko**
This is based on #465 PR. All changes are in one commit now.
Configuration details are in `docs/configuration.rst` file.

### CODE REVIEW

File: src/Phinx/Config/Config.php [hunk 1]
```diff
@@ -153,6 +153,49 @@ public function getEnvironment($name)
...
+                if ($database != $environments[$name]['name']) {
+                    throw new \InvalidArgumentException(sprintf(
+                        'The database "%s" does not exists in environment "%s"',
+                        $database,
+                        $name
+                    ));
+                }
+                $environments[$name]['name'] = $database;
+            }
+            elseif (!empty($environments[$name]['databases']) && $database !== null ) {
```
Comment (shadowhand) [inline src/Phinx/Config/Config.php]: But if `$database = false`, is it still okay?

Comment (bartekrylko) [inline src/Phinx/Config/Config.php]: Moslt likely not. Proper value is verified in next condition.
`$database !== null` is in condition to verify if parameter was sent, or default value is provided.
Imho no need to check if `$database === false` but if you want, i can add it.

**Comment by bartekrylko** [comment]
hello, do we know something more here?

**Comment by rlx123** [comment]
Hey guys, any news on multidb support ? I don't conceal the fact that it will be very useful.

File: src/Phinx/Config/Config.php (hunk 1 above)
Comment (shadowhand) [inline src/Phinx/Config/Config.php]: I would recommend using `!empty($database)` instead.

**Comment by shadowhand** [comment]
Thank you for contributing! The branch you made this PR against is no longer accepting patches. This probably happened because we didn't merge your changes fast enough. :sob:

Please create a new PR against the current default branch if you want to see these changes merged.

**Pull request closed without merging**